### PR TITLE
Issue #6068: mention pocicli features in xdoc

### PR DESCRIPTION
**Pull request by romani**
Issue #6068

### CODE REVIEW

File: src/xdocs/cmdline.xml.vm [hunk 1]
```diff
@@ -167,6 +167,13 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         specifying a property file using the <code>-p</code> option.  If a
         property file is specified, the system properties are ignored.
       </p>
+      <p>
+        CLI is implemented by mean of <a href="https://picocli.info">picocli</a>
```
Comment (rnveach) [inline src/xdocs/cmdline.xml.vm]: `mean of` => `means of`.

File: src/xdocs/cmdline.xml.vm [hunk 2]
```diff
@@ -167,6 +167,13 @@ java -D&lt;property&gt;=&lt;value&gt;  \
...
+        library, so our CLI supports definition of arguments in file
```
Comment (rnveach) [inline src/xdocs/cmdline.xml.vm]: `library, so our CLI` => `library. Our CLI`

File: src/xdocs/cmdline.xml.vm (hunk 1 above)
Comment (romani) [reply src/xdocs/cmdline.xml.vm]: done.

File: src/xdocs/cmdline.xml.vm (hunk 2 above)
Comment (romani) [reply src/xdocs/cmdline.xml.vm]: done

**Comment by romani** [comment]
moved to #6182

**Pull request closed without merging**